### PR TITLE
Fix build errors on ubuntu 14.10

### PR DESCRIPTION
**Pull request by kamino**

### CODE REVIEW

File: libqpol/policy_extend.c [hunk 1]
```diff
@@ -868,9 +868,9 @@ static int qpol_syn_rule_table_insert_sepol_avrule(qpol_policy_t * policy, qpol_
 	ebitmap_destroy(&source_types2);
 	ebitmap_destroy(&target_types);
 	ebitmap_destroy(&target_types2);
-	return 0;
+	return error;
 
-      err:
```
Comment (pebenito) [inline libqpol/policy_extend.c]: Please don't change the indentation.

Comment (kamino) [inline libqpol/policy_extend.c]: Actually, this label was indented using spaces whereas the whole file is indented using tabs. May be this typo deserves a separate commit ? Anyaway, it is not that important.

**Comment by kamino** [comment]
I will revert the changes for the setools/policyrep/qpol.i file (fail label). Which workflow do you prefer for that ?

1/ I add a new commit to existing ones to take into account your recommandations, or
2/ I rewrite the history of my commits and make the changes on the faulty commit (thus, no new commit will be added but this will probably alter the comments on the pull request)

**Comment by pebenito** [comment]
In this case, I'd prefer the history rewrite. Did you run all of the unit tests (both for Python 2 and 3)?

**Comment by kamino** [comment]
On python 2.7 and python 3.4, the 184 tests ran without errors. BTW, have you thought about putting in place a travis-ci ? I can make a PR for that if you are interested.

**Comment by pebenito** [comment]
Travis-ci is new to me, so I will have to look more in to it, but I'm leaning towards using it, if you want to put a PR together.
